schemas/1.2/dataType.json

```diff
@@ -113,6 +113,7 @@
                                         "iso-date",
                                         "iso-date-future",
```

Comment:
Does this mean we should be removing `iso-date` or just clarifying that it operates in `Z`?